Mathlib/RingTheory/TwoSidedIdeal/IsSimpleRing.lean

```diff
@@ -0,0 +1,103 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+
+import Mathlib.RingTheory.TwoSidedIdeal.Lattice
```

Comment:
I know I have extreme views on this, but I would split out a `Defs` file with only this import, and then the results in another file. 

One should be able to talk about a simple ring without knowing `IsField`.